README.md

```diff
@@ -111,14 +114,14 @@ In terms of the three axes, `EventTarget` is:
 
 The `HTMLMediaElement` class is the base class that both `<audio>` and `<video>` elements extend from, and it adds a ton of API surface for controlling media playback.
 In terms of the three axes, `HTMLMediaElement` is:
-1. **Level of coupling**: Coupled. `HTMLMediaElement` is generally developed by the same entity as the classes that extend it.
+1. **Degree of coupling**: Coupled. `HTMLMediaElement` is generally developed by the same entity as the classes that extend it.
```

Comment:
I suggested "degree" instead of "level" because level seemed ambiguous: is it talking about depth (sub-classing and inheritance is very prevalent here, so could be) or amount.

Reply:
Yes, I agree "degree" works a lot better.